admin/data/sql/migrations/027.do.create-pupil-restart-table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE [mtc_admin].[pupilRestart] (
+  id int IDENTITY (1,1) NOT NULL,
+  pupil_id int NOT NULL,
+  createdAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  recordedByUser nvarchar(50) NOT NULL,
+  reason nvarchar(50) NOT NULL,
```

Comment:
this breaks referential integrity.  You should be recording the id of the record from pupilRestartCode table